MinecraftClient/Protocol/Handlers/PacketTypes/Palettes/PacketPalette111.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MinecraftClient.Protocol.Handlers.PacketTypes.Palettes
+{
+    /// <summary>
+    /// For Minecraft version 1.10 - 1.11.2
+    /// </summary>
+    class PacketPalette111 : PacketTypePalette
+    {
+        private List<PacketTypesIn> typeIn = new List<PacketTypesIn>()
+        {
+            PacketTypesIn.SpawnEntity,
```

Comment:
Currently those palette are created by hand. I think it is painful to hardcode every packet ID into the code so I decided to use index for replacement. But without the ID clearly stated it might make more mistakes is a fact. So I added a dumper method to dump out the packet ID for debugging.

Reply:
Seems like it is easy to generate a packet palette with ID in the source code after creating the packet type list. Just need to write a script for doing that.
i.e. from List
```cs
{
    PacketTypesIn.SpawnEntity,
    ...
}
```
to Dict with ID
```cs
{
    { 0x00, PacketTypesIn.SpawnEntity },
    ...
}
```

Reply:
I wrote a method to convert the existing packet List to the packet Dictionary with IDs hard-coded. New Dict will look like this:
![image](https://user-images.githubusercontent.com/39955851/91291647-f3063580-e7c7-11ea-93a9-93be882496b9.png)
If this is okay, I will convert all palette into this format.